lib/alephant/broker/component_meta.rb

```diff
@@ -21,6 +21,18 @@ def component_key
         "#{id}/#{opts_hash}"
       end
 
+      def as_json
```

Comment:
I'm not sure `as_json` is the best name for this. I'm thinking something that describes what the data actually represents rather than the format its expected in